### ISSUE TITLE
http-flv or websocket

### ISSUE THREAD

**Issue by angwangiot** [Roverr/rtsp-stream#21]
Hi, do you have any plans to convert RTSP to http-flv or websocket?

**Comment by Roverr** [comment]
Hi @WangAngIOT,

Interesting idea, I will take a closer look at it this week!

**Comment by WangAngIOT** [comment]
@Roverr  I found two projects that could do this.You can get some of that.

**Comment by Roverr** [comment]
Okay I've taken a look at them. Since it modifies the whole structure it will take some time to implement but I'll take a shot at it. Hopefully I can finish it by next week.

**Comment by Roverr** [comment]
So I made a couple of tests with this but I did not really have the time to finish it.

On the other hand this functionality seems like a bit out of scope for this project as it aims to solve the urgent issue of not being able to play an RTSP stream in a browser. For example, using these to put together quick demos for investors can be viable idea. But in the long run this will never have the feature set needed for everything as it does not try to solve everything. Also at the end of the day even the applications mentioned below can do this, you just have to dockerise them.
